meson_options.txt

```diff
@@ -37,4 +37,5 @@ option('enable-tizen-sensor', type: 'boolean', value: false)
 option('enable-edgetpu', type: 'boolean', value: false)
 option('enable-openvino', type: 'boolean', value: false)
 option('enable-vivante', type: 'boolean', value: false)
+option('enable-vd-product', type: 'boolean', value: false)
```

Comment:
let's name it ```enable-tizen-permission-check```, with default value: true.
For VD, use ```-Denable-tizen-permission-check=false```

Reply:
Then, I will add two options.
* enable-tizen-permission-check
* enable-tizen-feature-check